tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -19,8 +19,11 @@ def __init__(self, marionette):
         self.apps = GaiaApps(self.marionette)
         self.accessibility = Accessibility(self.marionette)
         self.frame = None
-        self.manifest_url = hasattr(self, 'manifest_url') and self.manifest_url or None
```

Comment:
If you really thin this can go, then I guess this line can be removed too?
         self.entry_point = hasattr(self, 'entry_point') and self.entry_point or None